platform-sdk/swirlds-benchmarks/build.gradle.kts

```diff
@@ -54,5 +54,6 @@ tasks.register<JMHTask>("jmhReconnect") {
 
     resultsFile.convention(layout.buildDirectory.file("results/jmh/results-reconnect.txt"))
 
-    benchmarkParameters.put("numRecords", listProperty("5000000"))
+    benchmarkParameters.put("numRecords", listProperty("100000"))
```

Comment:
Really commenting a few lines up, line 53...

Have you tried a large heap (32G or 64G, or more, and set both Xms and Xmx) with a different GC, such as the ZGC we use in production?  The default G1 GC may struggle with large heap sizes.
Here is the option set we nominally use in production (NMT may be slightly different, but this is what's "requested").
`-Xmx100g -Xms32g -XX:+UnlockExperimentalVMOptions -XX:+UseZGC -XX:ConcGCThreads=14 -XX:ZMarkStackSpaceLimit=16g -XX:+UseLargePages -XX:MaxDirectMemorySize=8g -cp swirlds.jar:data/lib/* com.swirlds.platform.Browser > /opt/hgcapp/services-hedera/HapiApp2.0/stdout.log 2>&1
`

Reply:
Thanks for the suggestion. I'm trying

`jvmArgs.set(listOf("-Xmx32g", "-Xms32g", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseZGC", "-XX:ConcGCThreads=5", "-XX:+UseLargePages", "-XX:MaxDirectMemorySize=8g"))`

and the JVM just comes to silence during the warm-up iteration after generating the state, and no progress is happening at all. Looks like a thread executor dies somewhere silently.

I'm unsure if this is the result of a merge from upstream that I did just before publishing the PR, or the result of the new JVM parameters. I'll be playing with this next week.

For now, the fix stands as it is in this PR.